app/controllers/event_disavowal_controller.rb

```diff
@@ -10,13 +10,13 @@ def new
       success: true,
       extra: EventDisavowal::BuildDisavowedEventAnalyticsAttributes.call(disavowed_event),
     )
-    analytics.event_disavowal(**result.to_h)
+    analytics.event_disavowal(user_id: disavowed_event.user.id, **result.to_h)
```

Comment:
I would suggest using `user_id` over `user.id`, in case the associated user record hasn't yet been loaded from the database. The former would be able to use the ID without querying for the user.

(Same note applies below)

```suggestion
    analytics.event_disavowal(user_id: disavowed_event.user_id, **result.to_h)
```

Reply:
Does that association come in from here?
https://github.com/18F/identity-idp/blob/3a4e861c169330519ada801520e3d25e2ee281e5/app/services/user_event_creator.rb#L118-L124

Reply:
Yeah, the event would be linked to the user. Both `user_id` and `user.id` achieve the same effect, the difference being that `user_id` won't need to query the full user object.